_docs/tasks/installing-istio.md

```diff
@@ -70,7 +70,11 @@ default namespace. They can be modified for deployment in a different namespace.
    ```
    * If the command displays an error, or does not display anything, it means the cluster does not support RBAC, and you can proceed to step 5 below.
    
-   * If the command displays 'beta' version, or both 'alpha' and 'beta', please apply istio-rbac-beta.yaml configuration:
+   * If the command displays 'beta' version, or both 'alpha' and 'beta', please apply istio-rbac-beta.yaml configuration as show below:
+   
+   *(Note: If you deploy Istio in another namespace than the "default" namespace, replace the "namespace: default" line 
```

Comment:
would does `default` work inside a *italic section, `testing`, 1,2,3*

looks like it does, try ? (check how it looks like on your local rendered version)

also once this is in you would cherry pick it on the release-0.1